src/main/java/Group15/View/HomeScreenView.java

```diff
@@ -44,21 +49,16 @@ private static Pane createTitlePane()
         return titlePane;
     }
 
-    // Here we can display some sort of list of workout, maybe predefined, saved, random or idk
     private static Pane quickWorkoutPane()
     {
-        VBox quickWorkoutPane = new VBox();
+        HBox quickWorkoutPane = new HBox();
         quickWorkoutPane.setSpacing(20);
         quickWorkoutPane.setAlignment(Pos.CENTER);
         quickWorkoutPane.setPrefSize(640, 600);
         quickWorkoutPane.setMaxWidth(Region.USE_PREF_SIZE);
-        quickWorkoutPane.setBackground(new Background(new BackgroundFill(Color.YELLOWGREEN, null, null)));
-        Label quickWorkoutLabel = new Label("A list of some sort of workouts could be displayed " +
-                "here in the colored area. They could have a title and a description\nand be formatted in a way" +
-                "where we can have 5-6 workouts displayed.");
-        Label additionalComment = new Label("The color should not be green btw, it's just to show the area to use");
-        quickWorkoutPane.getChildren().add(quickWorkoutLabel);
-        quickWorkoutPane.getChildren().add(additionalComment);
+        Pane savedWorkoutsPane = createWorkoutsPane("Saved");
+        Pane popularWorkoutsPane = createWorkoutsPane("Popular");
+        quickWorkoutPane.getChildren().addAll(savedWorkoutsPane, popularWorkoutsPane);
```

Comment:
I cant comment further down, but the "New Workout" button should open the MuscleView instead. The "My Workout" button can open the WorkoutView

Reply:
i know but someone haven't merged that one yet :)

Reply:
Check again ;)